src/Symfony/Component/Ldap/Adapter/ExtLdap/Collection.php

```diff
@@ -84,7 +85,13 @@ private function initialize()
             return;
         }
 
-        $entries = ldap_get_entries($this->connection->getResource(), $this->search->getResource());
+        $con = $this->connection->getResource();
```

Comment:
The connection is always established before this code is executed.

Either the `Query` class creates the connection if non-existent (with default values), or it uses the existing connection.